warp/Network/Wai/Handler/Warp/Response.hs

```diff
@@ -7,6 +7,7 @@
 module Network.Wai.Handler.Warp.Response (
     sendResponse,
     sanitizeHeaderValue, -- for testing
+    --  Provided here for backwards compatibility.
     warpVersion,
```

Comment:
Question for the maintainers: should we deprecate this and remove it in a future version?

```haskell
{-# DEPRECATED warpVersion "some message" #-}
warpVersion = Network.Wai.Handler.Warp.Settings.warpVersion
```

Reply:
I don't have a strong opinion.